compose/bin/setup

```diff
@@ -64,8 +52,8 @@ bin/clinotty bin/magento setup:install \
   --session-save-redis-log-level=4 \
   --session-save-redis-db=2 \
   --search-engine=elasticsearch7 \
-  --elasticsearch-host=$ES_HOST \
-  --elasticsearch-port=$ES_PORT \
+  --elasticsearch-host=elasticsearch \
+  --elasticsearch-port=9200 \
```

Comment:
We'll want to back out this update so a user can override the Elasticsearch host/port with env vars.

Reply:
ok, I will recover this part.

But I think this setting is very common. 
If someone need change Elasticsearch host/port, change here is not enough.

docker-compose.yml (just guess, not test yet)
```
elasticsearch:
    image: markoshust/magento-elasticsearch:7.6.2-2
    ports:
      - "9200:9200" # this also need to change
      - "9300:9300"
    environment:
      - "discovery.type=single-node"
      - ES_HOST=elasticsearch # add new value
      - ES_PORT=9200               # add new value
```

Reply:
Yep, I'd avoid the vars being set here though and set defaults within the bash file (see https://github.com/markshust/docker-magento/pull/384#discussion_r570235980). But, this is how & where someone would override these vars. This would be useful to add to the documentation.

Reply:
I think this push request (commit) should only keep changes about `HEALTHCHECK`.
For other environmental variables, I will create another one for it.